docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -649,41 +650,44 @@ The following tables list the available media file information.
 
 | Metadata name    | Description                              |
 |----------------|----------------------------------------|
-| `Media ID`       | ID of the media content                  |
-| `File path`      | Path of the media content                |
-| `Display name`   | Display name of the media content        |
-| `Media type`     | Media type of the media content          |
-| `Mime type`      | MIME type of the media content           |
-| `Size`           | File size of the media content           |
-| `Added time`     | Time the media content was added to the database |
-| `Modified time`  | Last modification time of the media content |
+| `Id`             | ID of the media content                  |
+| `Path`           | Path of the media content                |
+| `DisplayName`    | Display name of the media content        |
+| `MediaType`      | Media type of the media content          |
+| `MimeType`       | MIME type of the media content           |
+| `FileSize`       | File size of the media content           |
+| `DateAdded`      | Time the media content was added to the database |
+| `DateModified`   | Last modification time of the media content |
 | `Timeline`       | Time the media content was created       |
-| `Thumbnail path` | Path of the stored thumbnail image of the media content |
+| `ThumbnailPath`  | Path of the stored thumbnail image of the media content |
 | `Description`    | Description of the media content         |
 | `Longitude`      | Longitude of the media content           |
 | `Latitude`       | Latitude of the media content            |
 | `Altitude`       | Altitude of the media content            |
 | `Rating`         | Rating of the media content              |
-| `Favorite`       | Favorite status of the media content     |
+| `IsFavorite`     | Favorite status of the media content     |
 | `Title`          | Title of the media content               |
-| `Storage type`   | Storage type of the media content        |
+| `StorageId`      | Storage ID of the media content          |
+| `IsDrm`          | The media is DRM-protected or not        |
+| `StorageType`      | Storage ID of the media content        |
```

Comment:
line 672: Check the description. (change to The storage type of the storage that the media is stored on?)
line 673: StorageId was duplicated.

Reply:
I'll fix it.